hubblestack/extmods/nova_v2_modules/pkg.py

```diff
@@ -0,0 +1,143 @@
+# -*- encoding: utf-8 -*-
+"""
+Nova module to check whether a package is installed on host machine
+
+Example yaml for check:
+check_unique_id:
+  description: 'sample description'
+  tag: 'ADOBE-00041'
+  implementations:
+    - filter:
+        grains: 'G@osfinger:CentOS*Linux-7'
+      module: pkg
+      checks:
+        - name: rsync
+          # Optional param: version
+          version: "3.1.2-10.el7"
+
+Here we are checking if "rsync" package is installed or not. We can have more than one package listed here
+
+Version Comparison:
+Optional param: "version" is to check the version of installed package
+Following operators are allowed in version comparison:
+>=, <=, >, <
+
+If version is given with no operator, Default is exact comparison.
+
+Regex Support
+We have support for '*'. Example: name: rpm*
+
+Note: In this case it might return multiple packages, and if you have given version comparison too.
+All matched packages must comply to that version string.
+
+You can give multiple packages in the yaml, under "checks" attribute.
+"""
+import logging
+import os
+import re
+
+from distutils.version import LooseVersion
```

Comment:
Loose version is outdated and will be deprecated in near future.
Please use version from packaging module.